script/sync-ghes/index.ts

```diff
@@ -51,11 +55,10 @@ async function checkWorkflows(
         ));
         const iconName: string | undefined = workflowProperties["iconName"];
 
-        const isBlankTemplate = workflowId === "blank";
-        const partnerWorkflow = workflowProperties.categories === null;
+        const isPartnerWorkflow = workflowProperties.creator ? partnersSet.has(workflowProperties.creator.toLowerCase()) : false;
 
         const enabled =
-          (isBlankTemplate || !partnerWorkflow) &&
```

Comment:
Why remove check for `isBlankTemplate`?

Reply:
When `categories==null` was introduced to filter out partner templates, "blank" template also got filtered out as it has categories==null. To retain it, the check was added. See conversation [here](https://github.com/actions/starter-workflows/pull/520/files#r428923084).

Now that we use the "creator" field to filter out partner templates, we do not need this check anymore.

Reply:
Got it